ratatui-core/src/layout/layout.rs

```diff
@@ -203,8 +203,8 @@ impl Layout {
     /// on my laptop's terminal (171+51 = 222) and doubling it for good measure and then adding a
     /// bit more to make it a round number. This gives enough entries to store a layout for every
     /// row and every column, twice over, which should be enough for most apps. For those that need
-    /// more, the cache size can be set with [`Layout::init_cache()`].
-    /// This const is unused if layout cache is disabled.
+    /// more, the cache size can be set with `Layout::init_cache()` (requires the `layout-cache`
+    /// feature). This const is unused if layout cache is disabled.
```

Comment:
I think it would be better to say "not defined" rather than "unused"

Or we can simply remove the statement: (it should be clear from the comment)

```suggestion
    /// more, the cache size can be set with `Layout::init_cache()` (requires the `layout-cache`
    /// feature).
```